spec/factories/news/item_factory.rb

```diff
@@ -74,5 +74,33 @@
         CONTENT
       end
     end
+
+    trait :with_unsafe_html do
+      unsafe_html = "<script>alert('Hello, world!')</script>"
+
+      title { "Title #{unsafe_html}" }
+
+      content do
+        <<~CONTENT
+          ## Second heading #{unsafe_html}
+
+          This is a paragraph
+        CONTENT
+      end
+    end
+
+    trait :with_safe_html do
+      safe_html = '<abbr>MFN</abbr>'
```

Comment:
Likewise I think this is a bit uncommon, I'd just copy the `safe_html` contents into the other two strings